src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -82,25 +82,37 @@ public Command parseCommand(String userInput) throws ParseException {
         case ClearCommand.COMMAND_WORD -> new ClearCommand();
         case FindCommand.COMMAND_WORD -> new FindCommandParser().parse(arguments);
         case ListCommand.COMMAND_WORD -> new ListCommand();
-        case ListWeddingsCommand.COMMAND_WORD -> new ListWeddingsCommand();
+        case ListWeddingsCommand.COMMAND_WORD, ListWeddingsCommand.COMMAND_KEYWORD -> new ListWeddingsCommand();
         case ExitCommand.COMMAND_WORD -> new ExitCommand();
         case HelpCommand.COMMAND_WORD -> new HelpCommand();
-        case CreateTagCommand.COMMAND_WORD -> new CreateTagCommandParser().parse(arguments);
-        case DeleteTagCommand.COMMAND_WORD -> new DeleteTagCommandParser().parse(arguments);
-        case EditWeddingCommand.COMMAND_WORD -> new EditWeddingCommandParser().parse(arguments);
-        case CreateWeddingCommand.COMMAND_WORD -> new CreateWeddingCommandParser().parse(arguments);
-        case DeleteWeddingCommand.COMMAND_WORD -> new DeleteWeddingCommandParser().parse(arguments);
+        case CreateTagCommand.COMMAND_WORD, CreateTagCommand.COMMAND_KEYWORD
+                -> new CreateTagCommandParser().parse(arguments);
+        case DeleteTagCommand.COMMAND_WORD, DeleteTagCommand.COMMAND_KEYWORD
+                -> new DeleteTagCommandParser().parse(arguments);
+        case EditWeddingCommand.COMMAND_WORD, EditWeddingCommand.COMMAND_KEYWORD
+                -> new EditWeddingCommandParser().parse(arguments);
```

Comment:
seems like there is a missing test case for the edit wedding shorthand command, could that be added in?

Reply:
Yes so `EditWeddingCommand` was not initially tested at all in the `AddressBookParserTest` so I added that in a separate PR along with testing of the keyword

Reply:
yep i just saw sorry!